pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "x4i"
+version = "1.0.4"
+dependencies = [
+    "numpy",
```

Comment:
It would be good to add minimal required versions for those dependencies here to ensure correct functionality across systems. What are the minimal versions this has been tested with?